app/assets/javascripts/sort.js

```diff
@@ -18,13 +18,13 @@ $(document).on('turbolinks:load', function(){
   var controller = $("body").data('controller');
   var action = $("body").data('action');
 
-  if(controller == "rooms" && action == "show" || controller == "rooms" && action == "update"){
+  if(controller == "rooms" && action == "show" || controller == "rooms" && action == "update" || controller == "users" && action == "recordings"){
```

Comment:
The only thing I am not sure about is this if condition. Does this mean that everytime we use the sorting we are going to add references to the place we are going to use it? It would not be berrer to have a function that is calles only when needed?

Reply:
Good point. Code will be changed so that the code will run any time there exists a table on the view